lib/components/viewers/route-viewer.js

```diff
@@ -176,20 +191,21 @@ class RouteViewer extends Component {
       const textColor = getMostReadableTextColor(routeColor, route?.textColor)
       const fill = vehicleIconHighlight === false ? null : textColor
 
+      setTimeout(() => this.setState({ fade: false }), 15)
```

Comment:
Move this action out of the render method. (You don't want to allocate resources during `render`, and who knows how many times `render` gets called.)